tutorials/focalboard-project-management/index.mdx

```diff
@@ -0,0 +1,219 @@
+---
+meta:
+  title: Project management for technical teams with Focalboard on Ubuntu Instances
+  description: This page shows how to setup a Focalboard project-management tool on Cost-Optimzed Instances running Ubuntu Linux
+content:
+  h1: Project management for technical teams with Focalboard on Ubuntu Instances
+  paragraph: This page shows how to setup a Focalboard project-management tool on Cost-Optimzed Instances running Ubuntu Linux
+tags: compute instance focalboard ubuntu project-management open-source
+hero:
+categories:
+  - compute
```

Comment:
ici on a compute en catégorie mais est-ce qu'on rajoute pas use-cases ?

Reply:
il y a build and maintain